tests.py

```diff
@@ -247,6 +256,107 @@ def test_active_article_task_200s(self):
             200,
         )
 
+    def build_proofing_comparison_dict(self, published, theme):
+        setting_handler.save_setting(
+            'general',
+            'journal_theme',
+            self.journal_one,
+            theme,
+        )
+        clear_cache()
+        self.client.force_login(self.typesetter)
+        if published:
+            url = reverse(
+                'article_view',
+                kwargs={
+                    'identifier_type': 'id',
+                    'identifier': self.article_in_typesetting.pk,
+                }
+            )
+        else:
+            url = reverse(
+                'typesetting_preview_galley',
+                kwargs={
+                    'article_id': self.article_in_typesetting.pk,
+                    'galley_id': self.galley.pk,
+                    'assignment_id': self.active_typesetting_task.pk,
+                }
+            )
+        response = self.client.get(url)
+        soup = BeautifulSoup(response.content, 'html.parser')
+        data = []
+        for included_element in soup.find_all(
+            class_='proofing-test-include'
+        ):
+            for excluded_element in included_element.find_all(
+                class_='proofing-test-exclude'
+            ):
```

Comment:
As per the review in https://github.com/BirkbeckCTP/janeway/pull/3471

Can we instead build either an exclusion or an inclusion list of node selectors (Using `id` attributes where it makes sense) rather than relying on a ghost class?